src/sentry/models/group.py

```diff
@@ -419,6 +425,16 @@ class Group(Model):
         ),
         db_index=True,
     )
+    substatus = BoundedPositiveIntegerField(
+        null=True,
+        default=None,
+        choices=(
+            (GroupSubStatus.ARCHIVED_UNTIL_ESCALATING, _("Archived until escalating")),
+            (GroupSubStatus.ONGOING, _("Ongoing")),
+            (GroupSubStatus.ESCALATING, _("Escalating")),
+        ),
+        db_index=True,
```

Comment:
I don't think we need an individual index on this field, we'll mostly be searching by status and substatus I think?